packages/core/package.json

```diff
@@ -30,6 +30,7 @@
   },
   "dependencies": {
     "@backstage/config": "^0.1.1-alpha.23",
+    "@backstage/core": "^0.1.1-alpha.23",
```

Comment:
Core should not have a dependency on core :grin:

Reply:
that's what I get from trying to copy from other parts of the codebase :D